ForecastWeather.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+import React, { Component } from 'react';
+
+import {
+  StyleSheet,
+  View,
+  FlatList,
+  Image,
+  Text
+} from 'react-native';
+import moment from "moment";
+import { apiKey_openweather_hourly } from './apiKey';
+
+class ForecastWeather extends Component {
+	/**
+	* Get day string from current time
+	* @param seconds from API
+	* @return day string
+	*/
+	formatDay = (sec) => {
+		let today = moment();
+		return today.to(moment(sec));
+	}
+
+	/**
+	* Get time string
+	* @param seconds from API
+	* @return time string
+	*/
+	formatTime = (sec) => {
+		return moment(sec).format('h A'); 
+	}
+
+	/**
+	* Get weather icons from icon URI
+	* @param icon ID from API
+	* @return icon URI
+	*/
+	getIcon = (icon_id) => {
```

Comment:
you can separate this function as common util function. try not to define the same function many places when you need to change something. it makes you get into hard situation.